dashboard/app/models/block.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: blocks
+#
+#  id          :integer          not null, primary key
+#  name        :string(255)
+#  level_type  :string(255)
+#  category    :text(65535)
+#  config      :text(65535)
+#  helper_code :text(65535)
```

Comment:
what's `helper_code`?

Reply:
It's extra javascript to be added to the helper library to support this block. E.g. if you add a block for `setTint(color)`, this would include the javascript definition of that function.